baseplate/context/redis.py

```diff
@@ -148,3 +151,62 @@ def __init__(self, trace_name, server_span, connection_pool,
     def execute(self, **kwargs):
         with self.server_span.make_child(self.trace_name):
             return super(MonitoredRedisPipeline, self).execute(**kwargs)
+
+
+class MessageQueue(object):
+    """A redis-backed variant of ``message_queue.MessageQueue``.
```

Comment:
doc nit: if you do `:py:class:`~baseplate.message_queue.MessageQueue`` sphinx will link the class and cut off the module prefixes to be shorter.